### PR TITLE
Added Watch integration to gameView

### PR DESCRIPTION
**Pull request by JunaidRajah**
This feature covers the watch companion app functionality.

### CODE REVIEW

File: UnmaskedWatch Extension/Controllers/HeroOneInterfaceController.swift [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    override func willActivate() {
+        heroOneLabel.setText(heroOneName)
+        if currentHeroOneImageURL != heroOneImageURL {
+            heroOneImage.loadImage(with: heroOneImageURL)
+            currentHeroOneImageURL = heroOneImageURL
+        }
+    }
+
+    @IBAction func pressedOption1Button() {
+        print("sending message")
```
Comment (bgouws) [inline UnmaskedWatch Extension/Controllers/HeroOneInterfaceController.swift]: Don't commit this print statement 😄

File: UnmaskedWatch Extension/Controllers/InterfaceController.swift [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        } else {
+            connectGroup.setHidden(false)
+        }
+    }
+    
+    func handleSession(_ session: WCSession,
+                       didReceiveMessage message: [String : Any],
+                       replyHandler: (([String : Any]) -> Void)? = nil) {
+        connectGroup.setHidden(true)
+        if let stat = message["Stat"] as? String {//**7.1
```
Comment (bgouws) [inline UnmaskedWatch Extension/Controllers/InterfaceController.swift]: Commented code here.

File: UnmaskedWatch Extension/Controllers/InterfaceController.swift [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+            self.statLabel.setText(stat)
+        }
+
+        if let currentScore = message["currentScore"] as? String {//**7.1
```
Comment (bgouws) [inline UnmaskedWatch Extension/Controllers/InterfaceController.swift]: and here

**Review by bgouws — CHANGES_REQUESTED** [review]
Just some mild clean up -otherwise looks great.

**Review by justinewright — APPROVED** [review]
Looks good

**Review by bgouws — APPROVED** [review]
Looks good man.

**Pull request merged**